jscomp/ml/typedecl.ml

```diff
@@ -479,7 +472,13 @@ let transl_declaration ~typeRecordAsObject env sdecl id =
           (match lbls_opt with
           | Some (lbls, lbls') ->
             check_duplicates lbls StringSet.empty;
-            Ttype_record lbls, Type_record(lbls', rep), sdecl
+            let optionalLabels =
+              Ext_list.filter_map lbls (fun lbl ->
+                  if has_optional lbl.ld_attributes then Some lbl.ld_name.txt else None)
+            in
+            Ttype_record lbls, Type_record(lbls', if unbox then Record_unboxed false
```

Comment:
formatting

Reply:
Auto formatting is turned off, how do you want me to format it? Just ad hoc as I please, or how do you typically do it in the files with formatting turned off?

Reply:
Ad hoc, until we manage to turn on formatting (some mess with macros which are not even necessary).